js/analytics_text.js

```diff
@@ -0,0 +1,54 @@
+
+function addAnalyticsTextBlock() {
+	// get the name of the current page
+	var p = window.location.pathname.split("/");
+	var resource = p[p.length - 1];
+
+	// if we are on the analytics page display the analytics text block
+	if (resource == "analytics") {
+		var guidance_text = '<div id="center-alert-info-id">' +
```

Comment:
(and leave `center-alert-info` as a class, but add `id="analytics-guidance"` to the div as well)